scripts/toolboxSrc/pushToCDN.ts

```diff
@@ -25,46 +30,78 @@ const pushClientAssetsToCDN = async () => {
   console.log(`⛅️ Uploaded ${dirEnts.length} client assets to CDN`)
 }
 
-const pushTemplatesToCDN = async () => {
+const pushServerAssetsToCDN = async () => {
```

Comment:
we were just pushing client assets, now we push server assets, too.
This is useful for things like:
- anonymousAvatar getting referenced in emails
- our logo's URL getting published in mattermost messages